spec/models/application_letter_spec.rb

```diff
@@ -76,15 +76,17 @@
 
   it "can not be updated if status is changed and application status is locked" do
     application = FactoryGirl.build(:application_letter)
+    application.event.acceptances_have_been_sent = true
+    application.event.rejections_have_been_sent = true
```

Comment:
that seems the same as using an event in execution phase.
Maybe we need another test case, with an event in selection phase & participant selection is locked?
And you should rename the "application status" in the test name